components/menu/index.tsx

```
@@ -5,84 +5,73 @@ import stylesFn from "@/styles/menu/MenuItem.styles";
-                  router.push(item.href);
-                }}
-              >
-                {item.title}
-              </Text>
-            </View>
+            <ProfileItemCard
+              item={item}
+              onPress={() => {
+                router.push(item.href);
+              }}
+            />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)